### PR TITLE
[Chore] Setup project

### LINKED ISSUE #16: [Chore] Setup project
## Why

Setup repo
 
## Acceptance Criteria

Create git repo for API site

### PR DESCRIPTION
**Pull request by luongvo**
## What happened 🤔
- Setup nodejs backend app
- Setup necessary components

## PoW (a.k.a Proof Of Work)💪

image

### CODE REVIEW

**Comment by luongvo** [comment]
Updated ✅

**Review by hoangnguyen92dn — APPROVED** [review]
🚀  In Nimble we trust ❤️

File: web/bin/www [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    ? 'Pipe ' + port
+    : 'Port ' + port;
+
+  // handle specific listen errors with friendly messages
+  switch (error.code) {
+    case 'EACCES':
+      console.error(bind + ' requires elevated privileges');
+      process.exit(1);
+      break;
+    case 'EADDRINUSE':
```
Comment (lydiasama) [inline web/bin/www:71]: `EACCES` and `EADDRINUSE`... What's that? 😂

**Review by lydiasama — APPROVED** [review]

File: web/bin/www (hunk 1 above)
Comment (luongvo) [reply web/bin/www:71]: @lydiasama you can check more info about them [here](https://nodejs.org/api/errors.html#errors_common_system_errors) 🤓

**Pull request merged**